dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/ContainerLoaderTest.java

```diff
@@ -0,0 +1,62 @@
+package com.dotcms.rendering.velocity.services;
+
+import com.dotcms.datagen.ContainerAsFileDataGen;
+import com.dotcms.datagen.SiteDataGen;
+import com.dotcms.repackage.org.apache.commons.io.IOUtils;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.containers.model.FileAssetContainer;
+import com.dotmarketing.util.PageMode;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+
+import static org.junit.Assert.assertTrue;
+
+public class ContainerLoaderTest {
+
+    @BeforeClass
+    public static void prepare () throws Exception {
+
+        //Setting web app environment
+        IntegrationTestInitService.getInstance().init();
+    }
+
+    /**
+     * Method to test: {@link ContainerLoader#writeObject(VelocityResourceKey)}
+     * when: the File Container's html is build
+     * Should: return a div with the File Comntainer's id
+     */
+    @Test
+    public void aaa() throws DotDataException, DotSecurityException, IOException {
```

Comment:
please rename this test

Reply:
done https://github.com/dotCMS/core/pull/17950/commits/e8ef923ad798cfa1c6428e2516f90183bcd40a4e#diff-90a1b8ea255c50a142e8987ff9b84049R37